### PR TITLE
SPCTR-3328: Fix - Post Carousal responsive columns issue in editor

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Post Carousal responsive columns issue in editor
https://brainstormforce.atlassian.net/browse/SPCTR-3328

### Screenshots

https://d.pr/v/eXDuN8

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Post Carousel - Resolved an issue in which the responsive columns would not work as intended in editor.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Post Carousel - Resolved an issue in which the responsive columns would not work as intended in the editor.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a very minor changelog update needed.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by adi3890 — APPROVED** [review]

File: src/blocks/post/post-carousel/blog.js [hunk 2]
```diff
@@ -129,15 +129,22 @@ function Blog( props ) {
 
 	const equalHeightClass = equalHeight ? 'uagb-post__carousel_equal-height' : '';
 
-	let slideToShow = columnsFallback;
-	if ( deviceType === 'Tablet' ) {
-		slideToShow = tcolumnsFallback;
-	} else if ( deviceType === 'Mobile' ) {
-		slideToShow = mcolumnsFallback;
+	let displayedSlides = columnsFallback;
```
Comment (adi3890) [inline src/blocks/post/post-carousel/blog.js:132]: WE can just declare it here as we are defining it in default condition

Comment (rajmohanreddy774) [reply src/blocks/post/post-carousel/blog.js]: Done.

**Pull request merged**